doc/tutorials/ann/ann.txt

```diff
@@ -230,13 +229,13 @@ int main()
   arma::mat predictionTemp;
   model.Predict(testData, predictionTemp);
 
-  /* 
```

Comment:
Sorry about all of the diff spam from the whitespace cleanups.

Reply:
Whitespace cleanups are awesome!  Thank you. :+1: